### PR TITLE
Add some comments for distribute_transpiler

### PR DESCRIPTION
**Pull request by panyx0718**

### CODE REVIEW

File: python/paddle/fluid/distribute_transpiler.py [hunk 1]
```diff
@@ -192,22 +194,24 @@ def transpile(self,
         self.trainer_id = trainer_id
         pserver_endpoints = pservers.split(",")
 
-        # step1
+        # step1: For large parameters and gradients, split them into smaller
```
Comment (gongweibao) [inline python/paddle/fluid/distribute_transpiler.py]: Line 154 has comments already.

File: python/paddle/fluid/distribute_transpiler.py [hunk 2]
```diff
@@ -812,6 +824,9 @@ def _get_lr_ops(self):
                 find_ops.append(op)
         # make a union find struct by the ops in default_main_program
         ufind = UnionFind(block.ops)
+
+        # TODO(panyx0718): If lr_ops connects with other training
```
Comment (typhoonzero) [inline python/paddle/fluid/distribute_transpiler.py]: We'll use SSA to replace the current ufind structure. For current deeplearning programs, it works fine.

File: python/paddle/fluid/distribute_transpiler.py [hunk 3]
```diff
@@ -278,6 +283,8 @@ def get_pserver_program(self, endpoint):
                 orig_var_name = v.name[:suff_idx]
             else:
                 orig_var_name = v.name
+            #TODO(panyx0718): Should this be put in the else block below? It's
```
Comment (typhoonzero) [inline python/paddle/fluid/distribute_transpiler.py]: Can put directly under `else`

File: python/paddle/fluid/distribute_transpiler.py (hunk 2 above)
Comment (panyx0718) [reply python/paddle/fluid/distribute_transpiler.py]: Removed this comment

File: python/paddle/fluid/distribute_transpiler.py (hunk 3 above)
Comment (panyx0718) [reply python/paddle/fluid/distribute_transpiler.py]: Done

**Pull request merged**